g11n-ws/modules/md-data-api-s3impl/src/main/java/com/vmware/vip/messages/data/conf/S3Client.java

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
Dose it make sense to create it file in common module and share with 'g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/S3Client.java'? seems the codes are duplicated.

Reply:
it doesn't work, because it need to inject object